### PR TITLE
Add failing test for issue #2333

### PR DESCRIPTION
**Pull request by albertfdp**
**Summary**

This PR tries to have a starting point to solve #2333. It reproduces the issue, so it can start a discussion on the solution.

screen shot 2017-02-24 at 13 43 08

**Test plan**

The test plan should be that this test passes.

### CODE REVIEW

**Comment by albertfdp** [comment]
I'm very happy to work on a fix for it, but I might need some guidance.

File: __tests__/commands/add.js [hunk 1]
```diff
@@ -314,6 +314,46 @@ test.concurrent('add with offline mirror', (): Promise<void> => {
   });
 });
 
+test.concurrent('add-then-install git+ssh from offline mirror', () : Promise<void> => {
```
Comment (bestander) [inline __tests__/commands/add.js]: can you make it `test.skip` now?
So that CI is green when we merge it

Comment (albertfdp) [inline __tests__/commands/add.js]: @bestander Sure, I just pushed it, thanks!

**Comment by bestander** [comment]
All right, great start!
@albertfdp, would be great to get a fix next.
Ping me if you need some help.
The starting place would probably be git-fetcher.js https://github.com/yarnpkg/yarn/blob/master/src/fetchers/git-fetcher.js#L29 where it uses offline-mirror

**Pull request merged**